crates/storage/codecs/src/alloy/optimism.rs

```diff
@@ -0,0 +1,92 @@
+//! Compact implementations for Optimism types.
+
+use crate::Compact;
+use alloc::{borrow::Cow, vec::Vec};
+use alloy_consensus::{Receipt, TxReceipt};
+use alloy_primitives::Log;
+use op_alloy_consensus::{OpDepositReceipt, OpReceipt, OpTxType};
+
+#[derive(Debug, Clone, PartialEq, Eq, Compact)]
+#[reth_codecs(crate = "crate")]
+struct CompactOpReceipt<'a> {
```

Comment:
I believe this is missing

https://github.com/paradigmxyz/reth/blob/9b7133ffded592feb5229588894af83c539c4a98/crates/optimism/primitives/src/receipt.rs#L448-L449